source/reference/footnotes.txt

```diff
@@ -0,0 +1,25 @@
+=========
+Footnotes
+=========
+
+Limit footnotes to tangential explanations of greater technical depth that build
+understanding but can be ignored.
+
+Use ``[#footnote-identifier]`` to mark the footnote location in the text, then add the 
```

Comment:
Suggestion: Lets convert this to a procedure on how to create a footnote -- here's my draft of [the hyperlink page](https://github.com/mongodb/docs-meta/pull/115) (I still need to make some formatting edits, but I think a procedure would simplify the process of whoever is trying to learn it)